patch_test_shadow_fork_2.py

```
@@ -0,0 +1,50 @@
+import re
+
+with open('tests/test_shadow_fork.py', 'r') as f:
```

Ruff (0.15.6)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)